doc/whats_new/v1.0.rst

```diff
@@ -295,6 +295,10 @@ Changelog
   :pr:`17169` by :user:`Dmytro Lituiev <DSLituiev>`
   and :user:`Julien Jerphanion <jjerphan>`.
 
+- |Fix| Fixed a bug in class:`feature_selection.VarianceThreshold`
+  when a variance threshold is negative.
```

Comment:
To properly link the class and a minor adjustment to the text:

```suggestion
- |API| Raises an error in :class:`feature_selection.VarianceThreshold`
  when the variance threshold is negative.
```

Looking at this again, I think this is more of an API change where we are performing more validation.